react/week3/todo-app/src/Timer.js

```diff
@@ -0,0 +1,17 @@
+import React, {useState} from 'react';
+
+const Timer = () => {
+const [timeValue, setTimeValue] = useState(0);
+
+setInterval(() => {
+    setTimeValue(timeValue+1)
+}, 1000);
```

Comment:
Your timer doesn't work exactly as it should! The problem is that because you're setting a new TimeValue (changing the Timer component state) every 1000ms the Timer component gets re-rendered, and therefore you keep setting new Intervals!
The trick is to include it in a useEffect hook, but remember to also return clearInterval, otherwise you'll likely end up with several Intervals running again.
If you need some more help, I can give you some more tips :)